skyportal/handlers/api/gcn.py

```diff
@@ -598,6 +639,54 @@ async def get(self, dateobs=None):
                     tag_subquery, GcnEvent.dateobs != tag_subquery.c.dateobs
                 )
 
+            if properties_filter is not None:
+                for prop_filt in properties_filter:
+                    prop_split = prop_filt.split(":")
+                    if not (len(prop_split) == 1 or len(prop_split) == 3):
```

Comment:
the API spec makes it seem like there must be 3 values after split. How can you get one?

Reply:
adding clarification to the spec.